src/openms/source/ANALYSIS/OPENSWATH/SpectrumExtractor.cpp

```diff
@@ -387,28 +392,42 @@ namespace OpenMS
       spectrum.getFloatDataArrays().resize(5);
       spectrum.getFloatDataArrays()[1].setName("score");
       spectrum.getFloatDataArrays()[1].push_back(score);
+      features[i].setIntensity(score); // The intensity of a feature is (proportional to) its total ion count. http://ftp.mi.fu-berlin.de/pub/OpenMS/develop-documentation/html/classOpenMS_1_1Feature.html
```

Comment:
let's add in the following:
`features[i].setMetaValue("avgSNR",avgSNR);`
`features[i].setMetaValue("log10_total_tic",log10_total_tic);`
`features[i].setMetaValue("avgFWHM",avgFWHM);`
`features[i].setMetaValue("inverse_avgFWHM",inverse_avgFWHM);`